### ISSUE TITLE
Trouble using Home Assistant Scrape functionality with /98 /99 commands

### ISSUE THREAD

**Issue by nielsonm236** [nielsonm236/NetMod-ServerApp#84]
Ron is trying to use Home Assistant "Scrape" functions to use REST commands to interact with the module. The issues were reported via YouTube, so I will paraphrase that interaction here, then continue resolution under this Issue Heading.
https://www.home-assistant.io/integrations/scrape/

Ron: ... is there a way to get the actual state of each relay via url?
Ron: OK so code 99 gives a string of states, will have to figure out how to code this...
Ron: I'm not using mqtt but command_line as follows:
switch:
  - platform: command_line
    switches:
      irrig_zone1:
        command_on: "curl -X GET http://192.168.1.4/30000/00"
        command_off: "curl -X GET http://192.168.1.4/30000/01"
        friendly_name: Zone 01
      irrig_zone2:
        command_on: "curl -X GET http://192.168.1.4/30000/02"
        command_off: "curl -X GET http://192.168.1.4/30000/03"
        friendly_name: Zone 02
      irrig_zone3:
        command_on: "curl -X GET http://192.168.1.4/30000/04"
        command_off: "curl -X GET http://192.168.1.4/30000/05"
        friendly_name: Zone 03
      irrig_zone4:
        command_on: "curl -X GET http://192.168.1.4/30000/06"
        command_off: "curl -X GET http://192.168.1.4/30000/07"
        friendly_name: Zone 04
      irrig_zone5:
        command_on: "curl -X GET http://192.168.1.4/30000/08"
        command_off: "curl -X GET http://192.168.1.4/30000/09"
        friendly_name: Zone 05
      irrig_zone6:
        command_on: "curl -X GET http://192.168.1.4/30000/10"
        command_off: "curl -X GET http://192.168.1.4/30000/11"
        friendly_name: Zone 06
      irrig_zone7:
        command_on: "curl -X GET http://192.168.1.4/30000/12"
        command_off: "curl -X GET http://192.168.1.4/30000/13"
        friendly_name: Zone 07
      irrig_zone8:
        command_on: "curl -X GET http://192.168.1.4/30000/14"
        command_off: "curl -X GET http://192.168.1.4/30000/15"
        friendly_name: Zone 08
I need to see if there is a way to do the same and get the state, if not then mqtt might be the way to go.

Ron: I managed to get the ascii code from the original board (remember I haven't re-programmed it yet with your code) doing the following:
  - platform: scrape
    resource: http://192.168.1.4/30000/99
    select: "p"
Basically I am retrieving whatever is in the url "http://192.168.1.4/30000/99" at the <p> tag.

Ron: In your code the status is at 192.168.1.4:8080/99 but it's inside the <body> tag. How can I add a <div> tag with an id into the code? In HA I'm trying to get the status using a scrape template...

**Comment by nielsonm236** [comment]
And now the continuation of the issue:
The /98 and /99 commands do not have any tags in the returned webpage ... just the 16 characters of pin state data. I'm not familiar with Scrape. Is it absolutely necessary to have a tag?
The way this was originally coded I did have the returned page surrounded by typical webpage tags ... then got feedback that no tags should be present. The user that made that request did not mention what tool was being used at the time. Hopefully you can find a way to extract the data without tags.

**Comment by nielsonm236** [comment]
I can close this already. Ron reports via YouTube that he has it working.
